point-of-sale-app/frontend/src/main/java/com/google/abmedge/frontend/FrontendController.java

```diff
@@ -123,8 +139,57 @@ public String home() {
   }
 
   @PostMapping(value = "/pay")
```

Comment:
Comment describing this endpoint, please! thanks!

Reply:
Added comments in e1f79d92056505a7dc37613265ecd51f11140481